src/proc/align.cpp

```diff
@@ -11,10 +11,30 @@
 #include "align.h"
 #include "stream.h"
 
+#ifdef RS2_USE_CUDA
```

Comment:
Is it possible a platform will have both CUDA & SSE3 enabled?
Please check because if yes, we will have to choose as this will not compile IMO

Reply:
@Nir-Az, the user cannot choose to run with or without "SSE3". If CPU processing is preferred, LRS should automatically detect and use "SSE3" wherever possible.

If "BUILD_WITH_CUDA" flag is enabled, CUDA implementations will be preferred. User cannot dynamically choose between CUDA or SSE3.

Reply:
Yes I know, just wanted to make sure that if the user build with Cuda, and gis CPU support SSSE,
He will include both headers.
We need to make sure the headers does not conflict.

Reply:
Headers are not conflicting. Anyway, updated them to avoid such confusions.